apollo-rx2-support/src/main/java/com/apollographql/apollo/rx2/RxJavaExtensions.kt

```diff
@@ -0,0 +1,109 @@
+@file:Suppress("NOTHING_TO_INLINE")
+@file:JvmName("KotlinExtensions")
```

Comment:
If we plan on introducing more kotlin extensions that aren't related to Rx, maybe we'd want to rename this? I don't see anything wrong with keeping the original filename of RxJavaExtensions.

Reply:
I've thought about that. It's not meant for Java users. It's just a friendly name if anybody discovers. Since it is inside `rx2` package, it is unique across modules.

I've looked at beat practices established by others. Both AutoDispose and Retrofit use the same name.